src/realm/link_view.cpp

```diff
@@ -118,8 +135,8 @@ size_t LinkView::do_set(size_t link_ndx, size_t target_row_ndx)
 {
     size_t old_target_row_ndx = to_size_t(m_row_indexes.get(link_ndx));
     size_t origin_row_ndx = get_origin_row_index();
-    m_origin_column.remove_backlink(old_target_row_ndx, origin_row_ndx); // Throws
-    m_origin_column.add_backlink(target_row_ndx, origin_row_ndx);        // Throws
+    m_origin_column->remove_backlink(old_target_row_ndx, origin_row_ndx); // Throws
+    m_origin_column->add_backlink(target_row_ndx, origin_row_ndx);        // Throws
```

Comment:
This change from `.` to `->` threw the end of line comments out of alignment here and in a few other places (this is why I dislike such comments). I'm not sure whether we want to try to keep them aligned.